generators/server/templates/src/main/java/package/service/_UserService.java

```diff
@@ -263,6 +269,9 @@ public void updateUser(String firstName, String lastName, String email, String l
                 <%_ if (databaseType == 'mongodb' || databaseType == 'cassandra') { _%>
                 userRepository.save(user);
                 <%_ } _%>
+                <%_ if (searchEngine == 'elasticsearch') { _%>
+                // userSearchRepository.save(user); // left out by design
```

Comment:
Sure.  Can you please help me understand why it is not needed and/or recommended?

Only the admin is calling userUpdate(UserDTO user) currently, does that have something to do with it?

It had resolved "User managment" page not getting updates in my code, though I really have my doubts now (will revisit).